### DISCUSSION TITLE
Update the Prisma client to support Interactive transactions

### DISCUSSION THREAD

**Discussion by mahd1ar** [keystonejs/keystone#8246] [Questions] [answered]
Today, Keystonejs uses Prisma client version 4.3.1 which does not support the interactive transaction. Prisma client 4.7.0 and above supports this feature.

[https://github.com/prisma/prisma/releases/tag/4.7.0](https://github.com/prisma/prisma/releases/tag/4.7.0)

**Comment by dcousens** [comment, marked as answer]
Please see https://github.com/keystonejs/keystone/discussions/8084 for database transactions, and for `context.prisma` support, know that we have upgraded to >4.7.0 many moons ago
